README.md

```diff
@@ -50,12 +89,11 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
-___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
-___: Defines the distance between the center of an element and the center of the adjacent element.
+margin  : Defines the distance between an element's border and adjacent elements' borders.
+outline  : Inserts a "wall" around an element.
+padding  : Defines the distance between an element's content and its border.
+margin  : Overlays a "wall" on top of an element.
+margin  : Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
These are the correct answers...

```
margin: Defines the distance between an element's border and adjacent elements' borders.
border: Inserts a "wall" around an element.
padding: Defines the distance between an element's content and its border.
___: Defines the width of an element.
outline: Overlays a "wall" on top of an element.
___: Defines the distance between the center of an element and the center of the adjacent element.
```

`margin` only applies to the first one you selected. `border` and `outline` can be easily confused, although to be honest you won't be using `outline` much, if at all.